src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -198,11 +198,19 @@ class DefaultStorageKeyAllocator
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
 
-    static StorageKeyName GlobalSceneCountKey() { return StorageKeyName::Formatted("g/scc"); }
-    static StorageKeyName FabricSceneDataKey(chip::FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/sc", fabric); }
-    static StorageKeyName FabricSceneKey(chip::FabricIndex fabric, uint8_t id)
+    // Scene Count in an Endpoint's SceneTable
+    static StorageKeyName EndpointSceneCountKey(EndpointId endpoint) { return StorageKeyName::Formatted("g/scc/e/%x", endpoint); }
+
+    // General data of all scenes ID belonging to a Fabric on a specific endpoint
```

Comment:
I don't understans this comment.  What's actually stored under this key?

Reply:
Reworded comment and added precisions.